iaso/tests/api/test_exportrequests.py

```diff
@@ -147,3 +150,23 @@ def test_exportrequests_create_ko_when_no_version(self):
                 + " {'demo': 'noversion'}",
             },
         )
+
+    def test_udpate(self):
+        self.client.force_authenticate(self.user)
+
+        self.build_instance(self.village_1, self.uuid(1), "201901")
+        self.build_instance(self.village_2, self.uuid(2), "201901")
+        ExportRequestBuilder().build_export_request(filters={"period_ids": "201901,201902"}, launcher=self.user)
+        export_request = m.ExportRequest.objects.first()
+
+        with patch.object(DataValueExporter, "export_instances", return_value=None):
```

Comment:
I'm not a mock enthusiast (sometimes you end up testing the mocks and not the production code)
I would have tried to use "responses" to mock a real "dhis2" error (ideally the one spotted in the diagnose report)

https://github.com/BLSQ/iaso/blob/a5ab6d855fb0013b16e608694134c7f7a280f069/iaso/tests/tests_aggregate_exporter.py#L402-L417